src/app/app.component.html

```diff
@@ -25,4 +25,12 @@
       <ion-router-outlet id="main-content"></ion-router-outlet>
     </div>
   </div>
+  <ion-footer *ngIf="currentSkin.footerTemplateName">
```

Comment:
As mentioned in other files, really this should be linked in a similar way to sidemenu settings, such as a `footerMenuDefaults` variable which reads from main config (which in turn is modified from skins)